core/embed/projects/unix/main.c

```diff
@@ -37,6 +37,7 @@
 #include <unistd.h>
 
 #include <io/display.h>
+#include <sec/secret.h>
```

Comment:
This is a nitpick: This include can be nested inside `#ifdef USE_TROPIC`.

Reply:
https://github.com/trezor/trezor-firmware/pull/4172/commits/07425d69411a20883084721064158b4c5a70e8e5#diff-85c6076cdbcf5611bd53eaaaecdf7d5bdbb77966adcf50fb0f75ae7f7ff3e82fR60